workloads/sample/sample_run.py

```diff
@@ -77,25 +77,28 @@
 # Add a prefix to the table names.
 workload.options.create_prefix = "table_"
 
-# Create one table every 30 seconds when the database size is less than 120 GB.
+# Targeted database size in GB.
+db_size_target_gb = 50
+
+# Create one table every 30 seconds until we have reached the targeted database size.
 workload.options.create_interval = 30
 workload.options.create_count = 1
-workload.options.create_trigger = 120 * 1024
-workload.options.create_target = 120 * 1024
+workload.options.create_trigger = db_size_target_gb * 1024
+workload.options.create_target = db_size_target_gb * 1024
 
-# Drop five tables every 90 seconds when the database size exceeds 120 GB. Stop
-# dropping tables when the database size goes below 80 GB.
+# Drop five tables every 90 seconds when the database size exceeds the targeted database size by too
```

Comment:
`Drop five tables every 90 seconds when the database size exceeds the target database size. Stop when the database size is below the target size margin. `

Reply:
Thank you, see 9fdd3bc15d454cadfd0c5a85f921db40c614a9d5